CMakeLists.txt

```diff
@@ -10,16 +10,27 @@ project (utf8proc C)
 # API version number (defined in utf8proc.h).
 # Be sure to also update these in Makefile and MANIFEST!
 set(SO_MAJOR 2)
-set(SO_MINOR 3)
-set(SO_PATCH 1)
+set(SO_MINOR 4)
+set(SO_PATCH 0)
```

Comment:
As the comment says, these numbers indicate ABI compatibility and don't follow the API version number exactly.  From 2.3 to 2.4 there was no change in ABI which is why the minor version stayed 3.

Reply:
Though it's probably easier for us to just match the API number so that we don't have this discussion repeatedly.